src/content/inferno/js/mobs/JalZek.ts

```diff
@@ -137,8 +149,6 @@ export class JalZek extends Mob {
   }
 
   attackIfPossible() {
-    this.attackDelay--;
```

Comment:
Is this not needed?

Reply:
I moved the tick-down into Unit since it seemed odd to be doing it in both Player and Mob